packages/embeds/embed-core/src/ModalBox/ModalBox.ts

```diff
@@ -68,17 +83,23 @@ export class ModalBox extends HTMLElement {
       return;
     }
 
-    if (newValue == "loaded") {
-      this.getLoaderElement().style.display = "none";
-    } else if (newValue === "started") {
+    if (newValue === "loading") {
+      this.open();
+      this.hideIframe();
+      this.getLoaderElement().style.display = "block";
+    } else if (newValue == "loaded" || newValue === "reopening") {
       this.open();
+      this.showIframe();
+      this.getLoaderElement().style.display = "none";
     } else if (newValue == "closed") {
       this.close();
     } else if (newValue === "failed") {
       this.getLoaderElement().style.display = "none";
       this.getErrorElement().style.display = "inline-block";
       const errorString = getErrorString(this.dataset.errorCode);
       this.getErrorElement().innerText = errorString;
+    } else if (newValue === "prerendering") {
```

Comment:
`prerendering` state that doesn't show the embed and does things hidden from sight.